src/sentry/api/endpoints/organization_events.py

```diff
@@ -353,91 +358,182 @@ def _data_fn(scopedDataset, offset, limit, query) -> dict[str, Any]:
                 on_demand_metrics_type=on_demand_metrics_type,
             )
 
+        def _dashboards_data_fn(scopedDataset, offset, limit, scoped_query, dashboard_widget_id):
+            try:
+                widget = DashboardWidget.objects.get(id=dashboard_widget_id)
+                does_widget_have_split = widget.discover_widget_split is not None
+                has_override_feature = features.has(
+                    "organizations:performance-discover-widget-split-override-save",
+                    organization,
+                    actor=request.user,
+                )
+
+                if does_widget_have_split and not has_override_feature:
+                    # This is essentially cached behaviour and we skip the check
+                    split_query = scoped_query
+                    if widget.discover_widget_split == DashboardWidgetTypes.ERROR_EVENTS:
+                        split_dataset = discover
+                        split_query = (
+                            f"({scoped_query}) AND !event.type:transaction"
+                            if scoped_query
+                            else "!event.type:transaction"
+                        )
```

Comment:
made this change because I noticed a bug where if scoped query is an empty string, it evaluates to:
`() AND !event.type:transaction`
and the `()` get added as a free text condition